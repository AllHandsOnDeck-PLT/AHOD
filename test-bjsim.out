Hello, and welcome to our Blackjack Simulator
First things first let us welcome our player for the evening
There are 5 cards in the hand. A Hit Request
R3
A 3 value not too bad
3
Another hit request
B2
Looks like we stop here for tonight. Time to see what the next cards were
R5
R3
R9
B9
