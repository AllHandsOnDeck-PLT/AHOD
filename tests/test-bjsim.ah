int i
series<Card> deck
int score 
when do void INTRO():
{ 
    do PRINT("Hello, and welcome to our Blackjack Simulator")
    do PRINT("First things first let us welcome our player for the evening")
} 
when do series<Card> CREATEDECK(series<Card> deck):
{ 
    series<string> types
    series<int> values 
    int value
    string type 
    Card card
    types = ["R3", "R9", "B9", "B2", "R3"]
    values = [3, 9, 9, 2, 3]
    for (i = 0; i < types.size(); i = i + 1):
    {
        type = types[i]
        value = values[i]
        card = Card(type, true,value)   
        deck.push(card) 
    }
    return deck
} 
main:
{
    Card card
    int currvalue
    deck = [Card("R5", true,5)]
    do INTRO()
    deck = do CREATEDECK(deck)
    do PRINT("There are 5 cards in the hand. A Hit Request")
    card = deck[deck.size()-1]
    score = 0 
    currvalue = card.value 
    do PRINT(card.type)
    do PRINT("A 3 value not too bad")
    score = score + currvalue
    do PRINT(score)
    deck.pop()
    do PRINT("Another hit request")
    card = deck[deck.size()-1] 
    do PRINT(card.type)
    deck.pop()
    do PRINT("Looks like we stop here for tonight. Time to see what the next cards were")
    for (i = 0 ; i < deck.size(); i = i + 1):
    {
        card = deck[i]
        do PRINT(card.type)
        /*do PRINT(card.faceup)*/
    }
}
