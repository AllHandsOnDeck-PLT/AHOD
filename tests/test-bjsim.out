Hello, and welcome to our Blackjack Simulator
First things first let us welcome our player for the evening
Stephen
There are 5 cards in the hand. A Hit Request
Card:
R3
Current Score of:
3
Another hit request
Card:
B2
Current Score of:
5
Looks like we stop here for tonight. Time to see what the next cards were
Card:
R6
That is a high value card
Could have had a higher score of: 
11
Card:
R5
That is a high value card
Could have had a higher score of: 
14
Card:
R9
That is a high value card
Good that you stopped otherwise you would have a score of: 
23
Card:
B9
That is a high value card
Good that you stopped otherwise you would have a score of: 
32
